src/iqm_client/iqm_client.py

```diff
@@ -234,11 +250,13 @@ class RunRequest(BaseModel):
         description='ID of the calibration set to use, or None to use the latest calibration set'
     )
     'ID of the calibration set to use, or None to use the latest calibration set'
-    qubit_mapping: Optional[list[SingleQubitMapping]] = Field(
+    qubit_mappings: Optional[BatchQubitMapping] = Field(
```

Comment:
```suggestion
    qubit_mappings: Optional[QubitMappingBatch] = Field(
```
to match `CircuitBatch`?

Reply:
Well, I had the same dilemma when naming the variables. The logic behind current naming is that `QubitMappingBatch` would just mean a batch of qubit mappings, but in reality it is a specific batch, namely a batch intended for the particular instance of circuit batch. So `Batch` in `BatchQubitMapping` refers to circuit batch, so it is intended to mean "qubit mappings for circuit batch" and not "batch of qubit mappings" )) I don't have any strong opinions on which naming to use, so if you have one, I can rename variables accordingly.